0000-pr-merge-requirements.md

```diff
@@ -0,0 +1,55 @@
+- Feature Name: PR Merge Requirements
+- Start Date: 2019-05-21
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Lay out the requirements to merge a PR
+
+# Motivation
+[motivation]: #motivation
+
+As we begin our working groups, to make sure that our community is all on the same page, we want to
+clearly define our requirements to merge a PR. This should benefit developers, reviewers and those
+merging the PRs.
+
+# Design
+[design]: #detailed-design
+
+All PR requirements
+  - Approval Reviews: 1 approval review from core team member OR
+                      1 approval review from captain of working group
+  - All tests pass.
+  - Cannot merge your own PR until 1 reviewer approves from defined list above that is not the author.
+
+Bug Fix PR requirements:
+  - Test Coverage: regression test written to cover bug fix.
+
+Feature PR requirements:
+  - Test Coverage: tests written to cover new feature.
+  - Release Notes: Add note in release notes of new feature for relative release.
```

Comment:
Ouch, that is new for me. Where are the release notes file?

Reply:
doc/topics/releases

Reply:
Thanks. Maybe is my fault, but I do not see this process documented.

If this is the case maybe we need to extend the page in saltstack that explain how to contribute, and add those pointers and the location of the releases file.

Reply:
thanks for pointing that out. I'll include that in the documentation when we document PR requirements in the contributor docs after this is merged.